src/test/java/org/broadinstitute/hellbender/tools/copynumber/utils/MergeAnnotatedRegionsByAnnotationIntegrationTest.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.copynumber.arguments.CopyNumberStandardArgument;
+import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedinterval.AnnotatedIntervalCollection;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MergeAnnotatedRegionsByAnnotationIntegrationTest extends CommandLineProgramTest {
+    private static final String TEST_SUB_DIR = publicTestDir + "org/broadinstitute/hellbender/tools/copynumber/utils/";
```

Comment:
Can use `toolsTestDir` to clean this up a bit.

Reply:
Done